src/components/App.jsx

```diff
@@ -14,8 +14,8 @@ const App = ({props}) => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path='/' element={<Login />} />
-        <Route path='board' element={<Board />} />
+        <Route path='/' element={<Board />} />
```

Comment:
Temporarily swapped board and login during development to easily route to the board without having to first type anything.